### PR TITLE
🐒Degen Mode

### PR DESCRIPTION
**Pull request by jvonasek**

### CODE REVIEW

File: src/components/CheckBox/CheckBox.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import { SContainer, SOuter } from "./CheckBox.styled"
+
+export type CheckboxSize = "small" | "medium" | "large"
+export type CheckboxVariant = "primary" | "secondary"
+export type CheckBoxProps = {
+  className?: string
+  label?: React.ReactNode
+  checked?: boolean
+  disabled?: boolean
+  onChange?: (checked: boolean) => void
```
Comment (vkulinich-cl) [inline src/components/CheckBox/CheckBox.tsx:11]: why onChange and checked props are not required?

File: src/components/Layout/Header/toolbar/buttons/Settings.tsx [hunk 2]
```diff
@@ -65,11 +101,27 @@ export const SettingsContents = ({ onClose }: { onClose: () => void }) => {
...
+              />
+              <HeaderDropdownItem
+                variant="toggle"
+                icon={<ApeIcon width={22} height={22} />}
+                title={t("header.settings.items.degenMode.title")}
+                subtitle={t("header.settings.items.degenMode.subtitle")}
+                tooltip={t("header.settings.degenMode.description")}
+                toggleValue={degenMode}
+                onClick={onDegenModeChange}
+                css={{ background: "transparent " }}
```
Comment (vkulinich-cl) [inline src/components/Layout/Header/toolbar/buttons/Settings.tsx:122]: there is a space in the string

**Comment by jak-pan** [comment]
Was doing some swaps and the degen mode assets disappear from trade form selector (need refresh screen to get them back)

File: src/components/CheckBox/CheckBox.tsx (hunk 1 above)
Comment (jvonasek) [reply src/components/CheckBox/CheckBox.tsx]: fixed

**Review by vkulinich-cl — APPROVED** [review]

**Comment by jak-pan** [comment]
Screenshot 2024-06-28 at 20 33 21

**Pull request merged**